src/instructlab/model/serve_backend.py

```diff
@@ -23,6 +28,55 @@ def signal_handler(num_signal, __):
     sys.exit(0)
 
 
+def get_gpu_count() -> int:
+    # Third Party
+    import torch
+
+    if torch.cuda.is_available():
+        return torch.cuda.device_count()
+    return 0
+
+
+def check_gpu_count(gpus: int, available_gpus: int) -> bool:
+    return gpus < available_gpus
```

Comment:
I believe this should be `gpus <= available_gpus`, right? Otherwise, we'll end up telling the user that if they have 4 GPUs and try to use all 4 GPUs that it's invalid?

Reply:
The fix looks good - resolving this thread. Thanks!